R/utils-arg.R

```diff
@@ -0,0 +1,62 @@
+handle_arg_list <- function(..., tests) {
+  values <- list(...)
+  names <- eval(substitute(alist(...)))
+  names <- purrr::map(names, deparse)
+
+  purrr::walk2(names, values, tests)
+}
+
+arg_is_scalar <- function(..., allow_null = FALSE, allow_na = FALSE) {
+  handle_arg_list(
+    ...,
+    tests = function(name, value) {
+      if (length(value) > 1 | (!allow_null & length(value) == 0)) {
+        cli::cli_abort("Argument {.val {name}} must be of length 1.")
+      }
+      if (!is.null(value)) {
+        if (is.na(value) & !allow_na) {
+          cli::cli_abort(
+            "Argument {.val {name}} must not be a missing value ({.val {NA}})."
+          )
+        }
+      }
+    }
+  )
+}
+
+arg_is_int <- function(..., allow_null = FALSE) {
+  arg_is_numeric(..., allow_null = allow_null)
```

Comment:
suggestion: This function doesn't exist, and is causing an issue with trying to use `.max_facets`. I assume this was supposed to be `arg_is_scalar`.

```
autoplot(jhu_csse_daily_subset, case_rate_7d_av, .facet_by = "geo_value", .max_facets = 3)
# Error in arg_is_numeric(..., allow_null = allow_null) : 
#   could not find function "arg_is_numeric"
```

```suggestion
  arg_is_scalar(..., allow_null = allow_null)
```

Reply:
No, this is correct. Adding the missing function.